samples/compose-resources-gallery/macosApp/build.gradle.kts

```diff
@@ -108,39 +108,39 @@ tasks.withType<KotlinNativeLink>()
 
 // TODO move to moko-resources gradle plugin
 // copy .bundle from .kexe to .app
-tasks.withType<org.jetbrains.compose.experimental.uikit.tasks.ExperimentalPackComposeApplicationForXCodeTask>()
-    .configureEach {
-        val packTask: org.jetbrains.compose.experimental.uikit.tasks.ExperimentalPackComposeApplicationForXCodeTask =
-            this
-
-        val kclass =
-            org.jetbrains.compose.experimental.uikit.tasks.ExperimentalPackComposeApplicationForXCodeTask::class
-        val kotlinBinaryField =
-            kclass.declaredMemberProperties.single { it.name == "kotlinBinary" }
-        val destinationDirField =
-            kclass.declaredMemberProperties.single { it.name == "destinationDir" }
-        val executablePathField =
-            kclass.declaredMemberProperties.single { it.name == "executablePath" }
-
-        @Suppress("ObjectLiteralToLambda") // lambda broke up-to-date
-        val action = object : Action<Task> {
-            override fun execute(t: Task) {
-                val kotlinBinary: RegularFile =
-                    (kotlinBinaryField.get(packTask) as RegularFileProperty).get()
-                val destinationDir: Directory =
-                    (destinationDirField.get(packTask) as DirectoryProperty).get()
-                val executablePath: String =
-                    (executablePathField.get(packTask) as Provider<*>).get().toString()
-
-                val outputDir: File = File(destinationDir.asFile, executablePath).parentFile
-
-                val bundleSearchDir: File = kotlinBinary.asFile.parentFile
-                bundleSearchDir
-                    .listFiles { file: File -> file.extension == "bundle" }
-                    ?.forEach { file ->
-                        file.copyRecursively(File(outputDir, file.name), true)
-                    }
-            }
-        }
-        doLast(action)
-    }
+//tasks.withType<org.jetbrains.compose.experimental.uikit.tasks.ExperimentalPackComposeApplicationForXCodeTask>()
```

Comment:
this commented code not required anymore? can we delete it?

Reply:
removed